datamodel_code_generator/parser/base.py

```diff
@@ -148,7 +151,10 @@ def add_ref(self, ref: str) -> Reference:
 
         name = self.get_class_name(original_name, unique=False)
         reference = Reference(
-            path=parents.split('/'), original_name=original_name, name=name
+            path=parents.split('/'),
+            original_name=original_name,
+            name=name,
+            loaded=not (ref.startswith('https://') or ref.startswith('http://'))
```

Comment:
```suggestion
            loaded=not ref.startswith(('https://', 'http://'))
```
iirc startswith support tuples.

Reply:
OK, I fix it too.